docs/upgrade/0.9.1_to_0.10.0.rst

```diff
@@ -0,0 +1,50 @@
+Upgrade from 0.9.1 to 0.10.0
+============================
+
+Updating the Tails Workstations
+-------------------------------
+
+We recommend that you update all Tails drives to version 3.10, which was released
+concurrently with SecureDrop 0.9.0 on October 23, 2018. Follow the Tails
+graphical prompts on your workstations to perform this upgrade.
+
+Due to a bug in the graphical SecureDrop updater that was fixed in SecureDrop
+0.9.1 (released on September 6, 2018), attempting an update of your SecureDrop
+workstation code on your *Journalist* or *Admin Workstations* using the
+graphical updater will fail with an error message: "WARNING: Signature
+verification failed."
```

Comment:
will fail -> may fail

I think there's potential for more refactoring here for clarity -- my personal bias would be to lead with the normal procedure (how to use the graphical updater), and then add the caveats (will not appear if running workstation code older than 0.7.0, may fail with error if running older than 0.9.1) & point to the docs as you do.

Reply:
Ack. I'll reach out for potential messaging in a DM @eloquence so as to avoid lots of commits.